plugins/com.google.cloud.tools.eclipse.appengine.ui/plugin.properties

```diff
@@ -1,9 +1,14 @@
 pluginName=Cloud Tools for Eclipse User Interface
 providerName=Google Inc.
-googleCloudPlatformAcronym=GCP   
-appEngineAcronym=GAE    
-googleName=Google   
 
-# TODO Why do we have both of these?    
+# Used for keywords
+googleCloudPlatformAcronym=GCP
+appEngineAcronym=GAE
+googleName=Google
 appEngineName=AppEngine   
-appEngineNameWithSpace=AppEngine
+appEngineNameWithSpace=App Engine
+googleCloudPlatformWithSpace=Google Cloud Platform
```

Comment:
I don't think we'll have this one without space, so probably just googleCloudPlatform or gogleCloudPLatformName